functions/src/handlers/workflow-cloud.ts

```diff
@@ -4,6 +4,8 @@ import { userHash } from '../utils/get-hash';
 
 const crypto = require('crypto');
 
+const UNKNOWN = 'Adapter name';
```

Comment:
That is an interesting name to give to a variable... hehe 
What is this variable? Maybe we find a bit more explanatory name? Like defaultAdapterName? Or AdapterPlaceholder? 
not sure if make sense in the context tho